src/lib/sectionCategories.ts

```diff
@@ -0,0 +1,25 @@
+/*!
```

Comment:
Let's call this `section-categories.ts`.

Reply:
Done.